src/App.tsx

```diff
@@ -1,9 +1,45 @@
+import { useEffect, useState } from 'react';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { getPeople } from './api/api';
 import './App.scss';
+import { Header } from './Components/Header';
+import { PeopleTable } from './Components/PeopleTable';
+import { Person } from './react-app-env';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+const App: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    getPeople()
+      .then(peopleFromServer => setPeople(peopleFromServer));
+  }, []);
```

Comment:
Checklist 7.Fetch people from [API](https://mate-academy.github.io/react_people-table/api/people.json) when PeoplePage is opened;

Reply:
missed that point, thanks